docs/docs/sourcing-from-woocommerce.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Sourcing from WooCommerce
+---
+
+WooCommerce is the e-commerce platform for WordPress. This guide will serve as a helpful reference whether you’re interested in getting started with e-commerce for your WordPress site or beginning to use Gatsby with your current WooCommerce setup.
+
+## Prerequisites
+
+This guide assumes some familiarity with WordPress. You may want to read about [Sourcing from WordPress](/docs/sourcing-from-wordpress/) or step through a [tutorial on how to build a blog with WordPress and Gatsby](/blog/2019-04-26-how-to-build-a-blog-with-wordpress-and-gatsby-part-1) before diving in.
+
+You’ll need a WordPress site with the [WooCommerce](https://woocommerce.com/) plugin installed and activated.
+
+## Existing plugins
+
+This guide assumes the use of the [`gatsby-source-woocommerce` plugin](/packages/@pasdo501/gatsby-source-woocommerce/). It is a straightforward solution specific to WooCommerce. 
+
+Install it:
+
+```bash
+npm install --save @pasdo501/gatsby-source-woocommerce
+``` 
+
+Configure its options:
+
+```js:title=gatsby-config.js
+{       
+  resolve: "@pasdo501/gatsby-source-woocommerce",
+  options: {
+    // Base URL of Wordpress site
+    api: 'wordpress.domain',
+    // true if using https. false if nah.
+    https: false,
+    api_keys: {
+      consumer_key: `your key`,
+      consumer_secret: `your secret`,
+    },
+    // Array of strings with fields you'd like to create nodes for...
+    fields: ['products', 'products/categories', 'products/attributes'],
+  }
+},
+```
+
+The consumer key and secret come from WooCommerce. From your dashboard, go to WooCommerce > Settings > Advanced > REST API and then add a key. You can then copy and paste both the key and secret. Make sure to store any sensitive information in [environment variables](/docs/environment-variables/)!
+
+There are other options which may make more sense for your project. For example, you may want to source more general site data like posts and pages from WordPress as well.
```

Comment:
Does this imply different set up? Different config?

Reply:
This is meant to lead into the next section. I could maybe move it down or remove it entirely if it's unclear.

Reply:
Ya, I might remove it. Don't think it's necessary for this guide. You have solid links for those use cases.